backupctl/history.py

```diff
@@ -20,6 +21,9 @@ def __init__(self, dbpath):
         self._conn = None
 
         self._path = dbpath
+        dirpath = os.path.dirname(self._path)
+        if not os.path.exists(dirpath):
+            os.makedirs(dirpath)
```

Comment:
Keeping this in history.py is ok for the first release (which we would like to get out of the door soonish) but will need refactoring to somewhere else once the history module isn't the only one using the db.